pallets/assets-registry/src/lib.rs

```diff
@@ -137,13 +131,23 @@ pub mod pallet {
 	}
 
 	// Convert MultiLocation to a Chainbridge compatible resource id.
-	pub trait IntoResourceId {
+	pub trait IntoResourceId<T: Get<Option<u8>>> {
```

Comment:
```suggestion
	pub trait IntoResourceId<T: Get<Option<u128>>> {
```

Easier to just match `GeneralIndex`

Reply:
Done